configure

```diff
@@ -2808,7 +2808,7 @@ fi
 
 # Define the identity of the package.
  PACKAGE='greasy'
- VERSION='2.2'
+ VERSION='2.2.3'
```

Comment:
I took the freedom to bump the version of the project.

Reply:
I can set to any value, just let me know.

Reply:
it is fine. Thanks for sharing your effort!